### PR TITLE
new notebooks

### PR DESCRIPTION
**Pull request by deborahduong**

### CODE REVIEW

File: covid19_sir/model/human.py [hunk 1]
```diff
@@ -185,6 +185,8 @@ def step(self):
             return
         if self.covid_model.current_state == SimulationState.EVENING_AT_HOME:
             self.disease_evolution()
+            if not self.is_infected() and not self.is_dead and flip_coin(0.0002):
```
Comment (andre-senna) [inline covid19_sir/model/human.py:188]: Is this change meant to be in this PR? It seems like a quick hack to test something.

**Review by andre-senna — CHANGES_REQUESTED** [review]

File: covid19_sir/model/human.py (hunk 1 above)
Comment (andre-senna) [reply covid19_sir/model/human.py:188]: @deborahduong 
I'll merge this PR and just comment out this line.

Comment (deborahduong) [reply covid19_sir/model/human.py:188]: @andre-senna Please put the line back in.  It is not a quick hack to test something it is a steady adding in of infections exogenously at  a low rate to show that the apparent herd immunity level was not due to stamping out all infections. thanks.

Comment (deborahduong) [reply covid19_sir/model/human.py:188]: Its actually two lines:  
            if not self.is_infected() and not self.is_dead and flip_coin(0.0002):
                    self.infect()

**Pull request merged**